docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/Skip.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Data;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    /// <summary>
+    /// Sample class showing how to use Skip.
+    /// </summary>
+    public static class Skip
+    {
+        public static void Example()
+        {
+            // Create a new context for ML.NET operations. It can be used for exception tracking and logging, 
+            // as a catalog of available operations and as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Get a small dataset as an IEnumerable.
+            IEnumerable<SamplesUtils.DatasetUtils.SampleTemperatureData> enumerableOfData = SamplesUtils.DatasetUtils.GetSampleTemperatureData(10);
+            var data = mlContext.Data.ReadFromEnumerable(enumerableOfData);
+
+            // Before we apply a filter, examine all the records in the dataset.
+            Console.WriteLine($"Date\tTemperature");
+            foreach (var row in enumerableOfData)
+            {
+                Console.WriteLine($"{row.Date.ToString("d")}\t{row.Temperature}");
+            }
+            Console.WriteLine();
+            // Expected output:
+            //  Date    Temperature
+            //  1/2/2012        36
```

Comment:
>36 [](start = 32, length = 2)

Those Americans pushing their unscientific temperature scale!
HAIL CELSIUS! VIVA LA KELVIN!

Reply:
This could be in Celsius, which is just Kelvin with an offset. Maybe in Australia.

---
In reply to: [254405704](https://github.com/dotnet/machinelearning/pull/2436#discussion_r254405704) [](ancestors = 254405704)